src/styles/variables.css

```diff
@@ -8,6 +8,7 @@
   --dark-blue: rgb(69, 85, 230);
   --sky-blue: rgb(90, 145, 255);
   --danger: rgb(248, 43, 101); /* user mentions color */
+  --banned: rgb(248, 43, 101, 0.2); /* banned tag bg color */
```

Comment:
you don't beed an extra css variable just for that, please remove.

You should use `color-mod` directly in the component css file, eg:
`fill: color-mod(var(--dark) alpha(70%));`

And it's --pink with 0.2, not danger

Reply:
Yup you're right olga thank you!